bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Flight/Profile.cpp

```diff
@@ -138,7 +143,19 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Flight_Profile(pybind11::module& aMo
             "target_velocity",
             &Profile::TrajectoryTarget::TargetVelocity,
             R"doc(
-                Create a target, which produces a vector pointing along the target velocity.
+                Create a target, which produces a vector pointing along the scan direction.
+                When choosing this as a clocking target, the resulting profile will not be yaw compensated.
```

Comment:
```suggestion
```
I would remove this line, I feel like "yaw compensated" can be an ambigious term